x-pack/filebeat/module/netflow/_meta/kibana/7/dashboard/filebeat-netflow-autonomous-systems.json

```diff
@@ -0,0 +1,597 @@
+            "size_y": 2
+          },
+          {
+            "col": 1,
+            "panelIndex": 5,
+            "panelRefName": "panel_4",
+            "row": 6,
+            "size_x": 6,
+            "size_y": 2
+          },
+          {
+            "col": 1,
+            "panelIndex": 6,
+            "panelRefName": "panel_5",
+            "row": 2,
+            "size_x": 4,
+            "size_y": 2
+          },
+          {
+            "col": 5,
+            "panelIndex": 7,
+            "panelRefName": "panel_6",
+            "row": 2,
+            "size_x": 4,
+            "size_y": 2
+          },
+          {
+            "col": 9,
+            "panelIndex": 8,
+            "panelRefName": "panel_7",
+            "row": 2,
+            "size_x": 4,
+            "size_y": 2
+          }
+        ],
+        "timeRestore": false,
+        "title": "[Filebeat] Netflow: Autonomous Systems",
+        "uiStateJSON": {},
+        "version": 1
+      },
+      "id": "c64665f9-d222-421e-90b0-c7310d944b8a",
+      "migrationVersion": {
+        "dashboard": "7.0.0"
+      },
+      "references": [
+        {
+          "id": "filebeat-*",
+          "name": "kibanaSavedObjectMeta.searchSourceJSON.filter[0].meta.index",
+          "type": "index-pattern"
+        },
+        {
+          "id": "d4e6520a-9ced-47c9-a8f2-7246e8cbd2d3",
+          "name": "panel_0",
+          "type": "visualization"
+        },
+        {
+          "id": "12aad647-c45d-4667-a029-152c1a97cbbc",
+          "name": "panel_1",
+          "type": "visualization"
+        },
+        {
+          "id": "d27b5d74-b3b4-4311-a0e6-08ff8f4345df",
+          "name": "panel_2",
+          "type": "visualization"
+        },
+        {
+          "id": "751ecb6f-11c3-458d-b039-f6d57a6379fa",
+          "name": "panel_3",
+          "type": "visualization"
+        },
+        {
+          "id": "f75063c7-48b7-4de4-b8cb-d07eb2cea0e9",
+          "name": "panel_4",
+          "type": "visualization"
+        },
+        {
+          "id": "f7808e70-df2a-4532-a350-966704567c24",
+          "name": "panel_5",
+          "type": "visualization"
+        },
+        {
+          "id": "aed09724-0a69-4331-84f5-3d2067c43930",
+          "name": "panel_6",
+          "type": "visualization"
+        },
+        {
+          "id": "f531f957-e8c0-497a-ad41-ef39c2d29671",
+          "name": "panel_7",
+          "type": "visualization"
+        }
+      ],
+      "type": "dashboard",
+      "updated_at": "2019-07-11T04:44:31.601Z",
+      "version": "WzM0MDMsMV0="
+    },
+    {
+      "attributes": {
+        "description": "",
+        "kibanaSavedObjectMeta": {
+          "searchSourceJSON": {
+            "filter": [],
+            "query": {
+              "query_string": {
+                "analyze_wildcard": true,
+                "query": "*"
+              }
+            }
+          }
+        },
+        "title": "Netflow: Dashboard Navigation [Filebeat]",
```

Comment:
It should be `Dashboard Navigation [Filebeat Netflow]` (same for other visualizations)